<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>654</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_configure">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>281</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_close">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_open">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>300</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_stop">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1.5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_serial">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>串口</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_stop">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_baudrate">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>75</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>110</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>134</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>150</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>300</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>14400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>56000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>76800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>128000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>256000</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_databit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>130</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_baudrate">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_databit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>67</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_crc">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_serial">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_crc">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>无</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>奇校验</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>偶校验</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_recv">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>321</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QTextEdit" name="textEdit_recv">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>321</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_recv">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>67</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>接 收</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_clear">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>清   空</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>112</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>16进制</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_send">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>641</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_send">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>20</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_send">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>521</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionversion"/>
   </widget>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionversion">
   <property name="text">
    <string>version</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
